PS_17144_DUST/YRC/YRC_17144.cpp

```diff
@@ -0,0 +1,97 @@
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+int R,C,T,conditioner;
+int dust[51][51];
+int dustcopy[51][51];
+int dx[4] = {1,0,-1,0};
+int dy[4] = {0,1,0,-1};
+
+void SpreadDust();
+void Cleaning();
```

Comment:
Inconsistency in naming is harming readability. 
How about just 'Clean()'?

And using capital letter in the first letter of name should be limited to names of Class.